src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -155,6 +155,12 @@
     "NAME": "Determination",
     "DESCRIPTION": "Remember stsjorbsmod:Pride. NL At the end of this turn, stsjorbsmod:Snap. NL Exhaust.",
     "UPGRADE_DESCRIPTION": "Remember stsjorbsmod:Pride. NL Exhaust."
+  },"stsjorbsmod:Discordance": {
```

Comment:
Formatting/indentation is a little inconsistent here (the localization files need to be understandable by non-programmers, so keeping the formatting consistent in them is especially important :) )

Reply:
Fixed